src/spellcheckquickfix.h

```diff
@@ -20,7 +20,11 @@
 
 #pragma once
 
+#if __has_include(<cppeditor/quickfixes/cppquickfix.h>)
+#include <cppeditor/quickfixes/cppquickfix.h>

```

Comment:
Is this due to a header relocation in Qt Creator 14?
At the start of the project I made the call that I will not support past Qt Creator versions, thus if Qt Creator moves the header can you please update the PR to use that header directly, without the additional complexity to work for older versions as well

Reply:
Ah, sorry for the mistake. Indeed, the move occurred in the 14th version. I changed the commit message.